docs/UserGuide.md

```diff
@@ -159,4 +159,4 @@ The `-p` or `--plugin` argument is used to provide a path to a BinSkim plugin th
 
 ## BinSkim Release History
 
-The latest version is always available on **[NuGet](https://www.nuget.org/packages/Microsoft.CodeAnalysis.BinSkim/)**. History is available on **[NuGet Release History](../src/ReleaseHistory.md)**.
+The latest version is always available on **[NuGet](https://www.nuget.org/packages/Microsoft.CodeAnalysis.BinSkim/)**. History is available on **[NuGet Release History](../ReleaseHistory.md)**.
```

Comment:
>ReleaseHistory [](http://example.com/codeflow?start=172&length=14)

Nice!

Reply:
this was actually already fixed in 4.1.0 
[Fix merge build error #858](https://github.com/microsoft/binskim/pull/858)

@schlaman-ms I think you can create a new PR so it is clean as one file change. (and some more change needed you mentioned)